manifests/4.3/local-volumes.crd.yaml

```diff
@@ -68,6 +68,45 @@ spec:
                   - devicePaths
                 type: object
               type: array
+            tolerations:
```

Comment:
We should define how we defined node selector here. The intent is to not redefine what definition already exists in Kubernetes.

Reply:
Understood. This has been updated to not replicate the K8s content in the list of defined items.